pySOT/tests/test_utils.py

```diff
@@ -73,9 +75,86 @@ def obj_function(x):
     x_best, f_best = ga.optimize()
     np.testing.assert_almost_equal(x_best[0], np.round(x_best[0]))
 
+POSITIVE_INFINITY = float("inf")
```

Comment:
Import the constant from `pySOT.strategy` or just remove it from `pySOT.strategy` We don't want to define the same constant twice.

Reply:
Done